mta/entities/report.py

```diff
@@ -1,21 +1,51 @@
 from widgetastic.utils import WaitFillViewStrategy
-from widgetastic.widget import Text
 from widgetastic.widget import View
 from widgetastic_patternfly import Tab
+from widgetastic_patternfly import Text
 
 from mta.entities import BaseLoggedInPage
-from mta.widgetastic import Input
+from mta.widgetastic import FilterInput
+from mta.widgetastic import MTADropdownMenu
 
 
 class AllApplicationsView(BaseLoggedInPage):
 
-    filter_by_name = Input(id="filter")
+    APP_ITEMS_LOCATOR = './/div[contains(@class, "fileName")]/a'
```

Comment:
It will be better to make an application list widget in widgetastic file for this , so we can use for other actions too .
WDYT ?

Reply:
Agree. I created new widget and used it in view.